src/app/shared/error/error.service.ts

```diff
@@ -62,6 +62,23 @@ export class ErrorService {
         concatMapTo(EMPTY)
       );
   }
+
+  toastDIABackendError$(err: unknown) {
```

Comment:
We had been treating acronym like a word in snake-case naming in all previous App codes (only capitalize the first character). It should be renamed to toastDiaBackendError$ for naming consistency.